study.md

```diff
@@ -87,7 +87,10 @@ Create the following movies with the given attributes.
 | 2 | The Core | 5.4 |
 
 ```ruby
-# your answer here
+movie = Movie.create(id: 0, title: "Battlefield Earth", rating: 2.4)
```

Comment:
All of the `id` keyword arguments should be omitted.  We never want to set `id`s ourselves when the database table has a primary key column to do it for us.